code/client/src/pages/document/view.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { RichTextEditor, Link } from '@mantine/tiptap';
+import { useEditor } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import './view.css';
+import {
+  BackgroundImage,
+  Card,
+  Text,
+  Button,
+  Group,
+  Box,
+  ActionIcon,
+  rem,
+  Tooltip,
+  CopyButton,
+  List,
+  ScrollArea,
+} from '@mantine/core';
+import { IconArrowLeft, IconCheck, IconDeviceFloppy } from '@tabler/icons-react';
+import { useClipboard } from '@mantine/hooks';
+
+const content =
+  '<h2 style="text-align: center;">Welcome to Mantine rich text editor</h2><p><code>RichTextEditor</code> component focuses on usability and is designed to be as simple as possible to bring a familiar editing experience to regular users. <code>RichTextEditor</code> is based on <a href="https://tiptap.dev/" rel="noopener noreferrer" target="_blank">Tiptap.dev</a> and supports all of its features:</p><ul><li>General text formatting: <strong>bold</strong>, <em>italic</em>, <u>underline</u>, <s>strike-through</s> </li><li>Headings (h1-h6)</li><li>Sub and super scripts (<sup>&lt;sup /&gt;</sup> and <sub>&lt;sub /&gt;</sub> tags)</li><li>Ordered and bullet lists</li><li>Text align&nbsp;</li><li>And all <a href="https://tiptap.dev/extensions" target="_blank" rel="noopener noreferrer">other extensions</a></li></ul>';
+
+export default function DocumentView() {
+  const editor = useEditor({
+    extensions: [StarterKit, Link],
+    content,
+  });
+
+  function ButtonCopy() {
+    const clipboard = useClipboard();
+  }
+
+  const docSaved = false;
```

Comment:
could you add comments on what this is?
so that when others read your code, they would understand the logic

Reply:
this was one that we are not using, i forgot to delete, will do